docs/qt.md

```diff
@@ -64,23 +107,35 @@ It is possible to build OpenDDS without using the configure script, although
 this is not recommended unless one is familiar with OpenDDS and it's build
 system, MPC. This list explains how the configure script tells MPC how to
 build Qt5 applications, but not how to build OpenDDS without the configure
-script in general.
+script in general. These environment variables can also be used with the
+configure script to override their values.
 
-  - The `qt5` feature must be enabled (e.g. `-features qt5=1`).
+  - The `qt5` feature must be enabled (e.g. `perl mpc.pl ... -features qt5=1 ...`).
```

Comment:
I see why this change is here but it seems like it could be more confusing than helpful.  When I run MPC the script I typically use is `mwc.pl` and I don't have a `perl` prefix since that's not necessary with the `#!` behavior on Linux and the file association on Windows.